macros/validation/validate_grain.sql

```diff
@@ -0,0 +1,13 @@
+{% macro validate_grain(grain, full_set) %}
```

Comment:
```suggestion
{% macro validate_grain(grain, metric_names) %}
```

Same comment as above